arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -1421,6 +1427,16 @@
 			path = "Raster layer (file)";
 			sourceTree = "<group>";
 		};
+		169CDBC8223C6A3A00ABFF0A /* Add a point scene layer */ = {
+			isa = PBXGroup;
+			children = (
+				169CDBC9223C6B5F00ABFF0A /* AddPointSceneLayer.storyboard */,
+				169CDBCB223C6B8400ABFF0A /* AddPointSceneLayerViewController.swift */,
+			);
+			name = "Add a point scene layer";
+			path = "New Group";
```

Comment:
This path is invalid. It may exist on your machine, but it doesn't get created when cloning/pulling because there are no files in it.

![Screen Shot 2019-03-19 at 8 59 47 AM](https://user-images.githubusercontent.com/2257493/54621687-61689500-4a25-11e9-9b20-965c97e5b1af.png)

The path should be the same as the name. You can set the path by tapping on the folder icon shown in the bottom right of the following screenshot:

![Screen Shot 2019-03-19 at 9 07 00 AM](https://user-images.githubusercontent.com/2257493/54622254-814c8880-4a26-11e9-805b-d565f925c065.png)

Reply:
I thought I fixed this, but the it worked only on my machine since file is there. Update the paths now